index.html

```diff
@@ -4,9 +4,21 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="/css/index.css">
-</head>
+    <link rel="stylesheet" href="	https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/explorer.css">
+  </head>
 <body>
-    
+    <section class="explorer">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-9"> 
```

Comment:
lets use `md` instead of `lg` for medium size devices, most of us use medium sizes? WDYT? 🤔